src/widgets/barchart/bar.rs

```diff
@@ -41,13 +41,35 @@ pub struct Bar<'a> {
 }
 
 impl<'a> Bar<'a> {
+    /// Creates a new ['Bar'] widget with the given label and value.
+    ///
+    /// The ```label``` parameter accepts any type that is convertible into a [`Line`].
+    /// The ```value``` parameter is the value to be displayed on the bar.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use ratatui::{prelude::*, widgets::*};
+    /// Bar::new("label", 10);
+    /// ```
+    pub fn new<L>(label: L, value: u64) -> Self
+    where
+        L: Into<Line<'a>>,
+    {
+        Self {
+            value,
+            label: Some(label.into()),
+            ..Default::default()
```

Comment:
I'd prefer being explicit here.
```suggestion
            style: Style::new(),
            value_style: Style::new(),
            text_value: None,
```